### DISCUSSION TITLE
🚀 FR - archive consent audit logs

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#4633] [Feature Requests]
### Suggest an idea for ExpressionEngine

I have a case where the consent_audit_log has well over 2 million records, it’s almost 40% of all the rows in the database. 

They do need to be able to have these logs, but it would be nice if there was a way to archive existing logs and basically clear out the database.  I'm not even certain what would happen if you backed up the table and then truncated it, which is what I would be tempted to do in their case.

But if we could have a way to do that, either just export and truncate the table or output to xml or whatever (which I can't see doing with that many records), it would be really helpful to big sites using consent.

**Comment by intoeetive** [comment]
Once we integrate Monolog https://github.com/ExpressionEngine/ExpressionEngine/pull/4041 they should be able to just use external DB or file to keep those
